x-pack/plugins/apm/public/components/app/service_dependencies/index.tsx

```diff
@@ -32,7 +32,7 @@ export function ServiceDependencies() {
         </EuiPanel>
       </ChartPointerEventContextProvider>
       <EuiSpacer size="l" />
-      <ServiceOverviewDependenciesTable />
+      <ServiceOverviewDependenciesTable hidePerPageOptions={false} />
```

Comment:
following my preview comment, here I'd do: 
```suggestion
      <ServiceOverviewDependenciesTable showPerPageOptions />
```